lib/install/ova/configure.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ova
+
+import (
+	"context"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25"
+	"github.com/vmware/govmomi/vim25/types"
+	"github.com/vmware/vic/pkg/errors"
+	"github.com/vmware/vic/pkg/vsphere/session"
+	"github.com/vmware/vic/pkg/vsphere/tasks"
+	virtualmachine "github.com/vmware/vic/pkg/vsphere/vm"
+)
+
+const (
+	// ManagedByKey defines the extension key to use in the ManagedByInfo of the OVA
+	ManagedByKey = "com.vmware.vic"
+	// ManagedByType defines the type to use in the ManagedByInfo of the OVA
+	ManagedByType = "VicApplianceVM"
+)
+
+func configureManagedByInfo(ctx context.Context, sess *session.Session, v *object.VirtualMachine) error {
+	vm := virtualmachine.NewVirtualMachineFromVM(ctx, sess, v)
+	spec := types.VirtualMachineConfigSpec{
```

Comment:
I'd also suggest applying this when the OVA is initially given the Admin credentials during Getting Started. This is when the OVA tags itself so that it can be located by the VCHs - it makes sense to also enable discovery by the plugin at the same time.

Performing this during Getting Started will also mean that multiple OVAs can easily be marked as managed by the extension without an extra user step and/or having to reinstall the plugin.

Reply:
@hickeng yeah that's our next step. I wanted to do that this sprint but we didn't have the time.